indico/modules/events/papers/blueprint.py

```diff
@@ -41,6 +41,8 @@
                  management.RHAssignmentListCustomize, methods=('GET', 'POST'))
 _bp.add_url_rule('/manage/papers/assignment-list/judge', 'manage_judge_papers',
                  management.RHBulkPaperJudgment, methods=('POST',))
+_bp.add_url_rule('/manage/papers/assignment-list/download', 'manage_download_papers', management.RHDownloadPapers,
```

Comment:
Are we going to need different handling for downloading in management and display? Maybe we don't need an endpoint specific to management.